python/ray/_raylet.pyx

```diff
@@ -1052,11 +1053,33 @@ cdef class CoreWorker:
         # Note: faster to not release GIL for short-running op.
         self.core_worker.get().RemoveObjectIDReference(c_object_id)
 
-    def promote_object_to_plasma(self, ObjectID object_id):
+    def serialize_object_id(self, ObjectID object_id):
         cdef:
             CObjectID c_object_id = object_id.native()
-        self.core_worker.get().PromoteObjectToPlasma(c_object_id)
-        return object_id.with_plasma_transport_type()
+            CTaskID c_owner_id = CTaskID.Nil()
+            CAddress c_owner_address = CAddress()
+        has_owner = self.core_worker.get().SerializeObjectId(
```

Comment:
s/SerializeObjectId/GetOwnershipInfo?

Reply:
I changed this to the wordier PromoteToPlasmaAndGetOwnershipInfo to make it clear that this is also mutating some state (to promote the object to plasma).